pkg/instance/helmfile.go

```diff
@@ -130,13 +130,14 @@ func configureInstanceEnvironment(accessToken string, instance *model.Instance,
 	cmd.Env = injectEnv(cmd.Env, "KUBERNETES_SERVICE_PORT_HTTPS")
 	cmd.Env = injectEnv(cmd.Env, "KUBERNETES_SERVICE_HOST")
 
-	for _, parameter := range instance.RequiredParameters {
-		instanceEnv := fmt.Sprintf("%s=%s", parameter.StackRequiredParameter.Name, parameter.Value)
-		cmd.Env = append(cmd.Env, instanceEnv)
-	}
-
-	for _, parameter := range instance.OptionalParameters {
-		instanceEnv := fmt.Sprintf("%s=%s", parameter.StackOptionalParameter.Name, parameter.Value)
+	for _, parameter := range stack.Parameters {
+		instanceParameter, err := instance.FindParameter(parameter.Name)
+		if err == nil {
+			instanceEnv := fmt.Sprintf("%s=%s", instanceParameter.StackParameter.Name, instanceParameter.Value)
+			cmd.Env = append(cmd.Env, instanceEnv)
+			continue
+		}
+		instanceEnv := fmt.Sprintf("%s=%s", parameter.Name, *parameter.DefaultValue)
```

Comment:
What if there is no default value? If whatever comes next can never succeed this should turn into an error.

Reply:
Why? Won't it just be nothing and that's ok?